packages/core/src/builtin-tasks/node.ts

```diff
@@ -4,20 +4,28 @@ import { CkbDockerNetwork } from '@ckb-js/kuai-docker-node'
 import { KuaiError } from '@ckb-js/kuai-common'
 import { ERRORS } from '../errors-list'
 import '../type/runtime'
+import { Indexer, RPC, config } from '@ckb-lumos/lumos'
+import { cachePath, configPath } from '../helper'
+import path from 'node:path'
+import fs from 'fs'
+import download from 'download'
 
 interface Args {
   port: number
   detached: boolean
   genesisArgs: Array<string>
+  filePath: string
 }
 
 const TERMINATION_SIGNALS = ['SIGINT', 'SIGTERM', 'SIGQUIT']
+const BUILTIN_SCRIPTS = ['anyone_can_pay', 'omni_lock', 'simple_udt']
 
 subtask('node:start', 'start a ckb node')
   .addParam('port', 'The port of the node', 8114, paramTypes.number)
   .addParam('detached', 'Run the node in detached mode', false, paramTypes.boolean)
+  .addParam('filePath', 'The script config file path', undefined, paramTypes.string, true, true)
```

Comment:
The `param` name could be more intuitive to indicate users it's about `script`, or it will be treated as `ckb node configuration`, e.g., `ckb.toml`